.github/workflows/build-test.yml

```diff
@@ -94,7 +94,12 @@ jobs:
         export HELM_NAMESPACE=ess
         for checkov_values in charts/matrix-stack/ci/*checkov*values.yaml; do
           echo "Testing matrix-stack with $checkov_values";
-          checkov -d "charts/matrix-stack" --framework helm --var-file "$checkov_values" --quiet
+          mkdir checkov-tmp
```

Comment:
Add to `.gitignore` or otherwise I think we should use `mktemp -d` so that we don't get a folder in this directory